pkg/cache/v2/listener_test.go

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2023 The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cache_v2
+
+import (
+	"testing"
+
+	"github.com/agiledragon/gomonkey/v2"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	core_v2 "kmesh.net/kmesh/api/v2/core"
+	listener_v2 "kmesh.net/kmesh/api/v2/listener"
+	maps_v2 "kmesh.net/kmesh/pkg/cache/v2/maps"
+	"kmesh.net/kmesh/pkg/utils/hash"
+)
+
+func TestListenerFlush(t *testing.T) {
```

Comment:
I am thinking you are testing `handleLdsResponse`, is that covered by those tests

Reply:
Yes, I'm testing if the listenercache is handled correctly when maps_v2.ListenerUpdate returns nil.
This covers the handleLdsResponse usage scenario.